homeassistant/components/sensibo/climate.py

```diff
@@ -419,3 +471,15 @@ async def api_call_custom_service_pure_boost(
         result = {}
         result = await self._client.async_set_pureboost(self._device_id, data)
         return bool(result.get("status") == "success")
+
+    @async_handle_api_call
+    async def api_call_custom_service_full_ac_state(
+        self,
+        key: str,
+        value: Any,
+        data: dict,
+    ) -> bool:
+        """Make service call to api."""
+        result = {}
```

Comment:
```suggestion
```
Leftover from testing?

Reply:
Probably as I wanted to force result into being a dict. I have removed it and will make another PR to remove it from the other calls.